dts/bindings/phy/st,stm32u5-otghs-phy.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2024 Meta
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    This binding is to be used by the STM32U5xx transceivers which are built-in
+    with USB HS PHY IP and a configurable HSE clock source.
+
+compatible: "st,stm32u5-otghs-phy"
+
+include: phy-controller.yaml
+
+properties:
+  "#phy-cells":
+    const: 0
+
+  clock-cfg:
+    type: int
+    enum:
+      - 1 # OTGHS_PHY_CLK_16MHZ
+      - 2 # OTGHS_PHY_CLK_19P2MHZ
+      - 3 # OTGHS_PHY_CLK_20MHZ
+      - 4 # OTGHS_PHY_CLK_24MHZ
+      - 5 # OTGHS_PHY_CLK_26MHZ
+      - 6 # OTGHS_PHY_CLK_32MHZ
```

Comment:
Please document values in the description to allow rendering in documentation.
See https://docs.zephyrproject.org/latest/build/dts/api/bindings/timer/st%2Cstm32-timers.html#dtbinding-st-stm32-timers as example